src/nepiada/utils/config.py

```diff
@@ -39,7 +43,9 @@ class Config:
     # Graph update parameters
     dynamic_obs: bool = True
     obs_radius: int = 5
-    full_communication: bool = True
+    dynamic_comms: bool = True
```

Comment:
When dynamic comms is turned ON, what is the qualitative result on convergence? Does the render show slow convergence or no convergence at all?

Reply:
From my testing, it depends on the minimum as well as the comms radius. If the minimum is low and comms radius is low, converge is either significantly slowed or doesn't converge at all (as expected). If the comms radius is sufficiently high compared to the world size convergence will usually happen albeit slower than full communication. If there's no minimum enforced its much more of a gamble, as if a drone spawns in a corner without anyone to communicate with, then it may just be stuck there